hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -617,6 +617,8 @@ profile::jenkinscontroller::jcasc:
               - maven-19-windows
   artifact_caching_proxy:
     disabled: false
+  datadog:
```

Comment:
Don't forget to add the plugin (same file, a list of plugins) `datadog` (I'm not sure if it is easy to make it automatic)